easybuild/easyconfigs/f/FusionCatcher/FusionCatcher-1.20-foss-2019b-Python-2.7.16.eb

```diff
@@ -0,0 +1,110 @@
+easyblock = 'Bundle'
+
+name = 'FusionCatcher'
+version = '1.20'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/ndaniel/fusioncatcher'
+description = """FusionCatcher searches for novel/known somatic fusion genes, translocations,
+and chimeras in RNA-seq data (paired-end or single-end reads from Illumina NGS
+platforms like Solexa/HiSeq/NextSeq/MiSeq/MiniSeq) from diseased samples."""
+github_account = 'ndaniel'
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+dependencies = [
+    ('Java', '11', '', True),
+    ('Python', '2.7.16'),
+    ('SciPy-bundle', '2019.10', versionsuffix),
+    ('Biopython', '1.75', versionsuffix),
+    ('BLAT', '3.5'),
+    ('Bowtie', '1.2.3'),
+    ('Bowtie2', '2.3.5.1'),
+    ('BWA', '0.7.17'),
+    ('Kent_tools', '20190326', '-linux.x86_64', True),
+    ('SRA-Toolkit', '2.10.4'),
+    ('STAR', '2.7.2b'),
+    ('picard', '2.21.6', '-Java-11', True),  # Optional
+    ('Velvet', '1.2.10', '-mt-kmer_191'),  # Optional
+    ('OpenPyXL', '2.6.4'),  # Optional
+    ('parallel', '20190922'),  # Optional
+    ('pigz', '2.4'),
+    ('zlib', '1.2.11'),
+]
+
+local_bbmap_makefile = 'makefile.%s' % {'Darwin': 'osx', 'Linux': 'linux'}[OS_TYPE]
+
```

Comment:
```
default_component_specs = {
    'sources': [{
        'download_filename': '%(version)s.tar.gz',
        'filename': '%(namelower)s-%(version)s.tar.gz',
    }],
    'start_dir': '%(namelower)s-%(version)s',
}

default_easyblock = "MakeCp"
```
Can be used to remove some cluttering in components.

Reply:
you are right, I'll put some common settings for the components of the bundle.

Reply:
I have added the `default_component_specs` from your easyconfig to this PR.